lib/engines/content_object_store/app/controllers/content_object_store/content_block/editions/steps_controller.rb

```diff
@@ -0,0 +1,74 @@
+class ContentObjectStore::ContentBlock::Editions::StepsController < ContentObjectStore::BaseController
+  VALID_STEPS = {
+    review_links: "review_links",
+    schedule_publishing: "schedule_publishing",
+  }.freeze
+
+  def show
+    step = params[:step]
+    @content_block_edition = ContentObjectStore::ContentBlock::Edition.find(params[:id])
+    @schema = ContentObjectStore::ContentBlock::Schema.find_by_block_type(@content_block_edition.document.block_type)
+
+    case step
+    when VALID_STEPS[:review_links]
+      review_links
+    when VALID_STEPS[:schedule_publishing]
+      schedule_publishing
+    end
+  end
+
+  def update
+    step = params[:step]
+    @content_block_edition = ContentObjectStore::ContentBlock::Edition.find(params[:id])
+    @schema = ContentObjectStore::ContentBlock::Schema.find_by_block_type(@content_block_edition.document.block_type)
+
+    case step
+    when VALID_STEPS[:review_links]
+      confirm_links
+    when VALID_STEPS[:schedule_publishing]
+      create_schedule
+    end
+  end
+
+private
+
+  def review_links
+    @content_block_document = @content_block_edition.document
+    @host_content_items = ContentObjectStore::GetHostContentItems.by_embedded_document(
+      content_block_document: @content_block_document,
+    )
+
+    render :review_links
+  end
+
+  def confirm_links
+    redirect_to content_object_store.content_object_store_content_block_step_path(id: @content_block_edition.id, step: :schedule_publishing)
+  end
+
+  def schedule_publishing
+    @content_block_document = @content_block_edition.document
+
+    render :schedule_publishing
+  end
+
+  def create_schedule
```

Comment:
is this method more like `schedule_or_publish`?

Reply:
Good call. I'll fix that 👍